config/logging.php

```diff
@@ -99,6 +99,10 @@
         'emergency' => [
             'path' => storage_path('logs/laravel.log'),
         ],
+
+        'bugsnag' => [
+            'driver' => 'bugsnag',
+        ],
     ],
 
 ];
```

Comment:
Can you also add `BUGSNAG_API_KEY` to the `.env.example` file?

Reply:
Done